playbook/actions/gitops.go

```diff
@@ -25,20 +24,22 @@ type GitOps struct {
 	shouldCreatePR bool
 }
 
+type Link struct {
+	ID   int    `json:"id,omitempty"`
+	Name string `json:"name,omitempty"`
+	URL  string `json:"url,omitempty"`
+	Icon string `json:"icon,omitempty"`
+}
+
 type GitOpsActionResult struct {
-	CreatedPR string `json:"createdPR,omitempty"`
-	Logs      string `json:"logs,omitempty"`
+	Links  []Link `json:"links,omitempty"`
+	Stdout string `json:"stdout,omitempty"`
```

Comment:
Lets keep it Logs to prevent breaking the FE, and have a `PR map[string]any` for the response from GH. 
I think it might also make sense to add a few more log lines, e.g.

2023-01-01 10:100:20  Creating PR on {repo}
Branch:
Commit:
Author: John Doe <john@doe.com>

2023-01-01 10:100:20 Created PR #123

Reply:
```json
{
    "pr": {
        "Ref": "refs/merge-requests/11/head",
        "Sha": "b7afbf2ec7b07043760c1dc06eeb14d412dc72cc",
        "Base": {
            "Ref": "main",
            "Sha": "",
            "Repo": {
                "ID": "54768975",
                "Link": "https://gitlab.com/flanksource-aditya/test",
                "Name": "test",
                "Perm": {
                    "Pull": true,
                    "Push": true,
                    "Admin": true
                },
                "Clone": "https://gitlab.com/flanksource-aditya/test.git",
                "Branch": "main",
                "Created": "2024-02-09T14:11:43.318Z",
                "Private": true,
                "Updated": "0001-01-01T00:00:00Z",
                "Archived": false,
                "CloneSSH": "git@gitlab.com:flanksource-aditya/test.git",
                "FullName": "flanksource-aditya/test",
                "Namespace": "flanksource-aditya"
            }
        },
        "Body": "",
        "Fork": "flanksource-aditya/test",
        "Head": {
            "Ref": "playbooks-branch-alertmanager-kube-prometheus-stack-alertmanager-2",
            "Sha": "b7afbf2ec7b07043760c1dc06eeb14d412dc72cc",
            "Repo": {
                "ID": "54768975",
                "Link": "https://gitlab.com/flanksource-aditya/test",
                "Name": "test",
                "Perm": {
                    "Pull": true,
                    "Push": true,
                    "Admin": true
                },
                "Clone": "https://gitlab.com/flanksource-aditya/test.git",
                "Branch": "main",
                "Created": "2024-02-09T14:11:43.318Z",
                "Private": true,
                "Updated": "0001-01-01T00:00:00Z",
                "Archived": false,
                "CloneSSH": "git@gitlab.com:flanksource-aditya/test.git",
                "FullName": "flanksource-aditya/test",
                "Namespace": "flanksource-aditya"
            }
        },
        "Link": "https://gitlab.com/flanksource-aditya/test/-/merge_requests/11",
        "Draft": false,
        "State": "open",
        "Title": "chore: Adding pod alertmanager-kube-prometheus-stack-alertmanager-2",
        "Author": {
            "ID": 20062246,
            "Link": "",
            "Name": "Aditya Thebe",
            "Email": "",
            "Login": "aditya5500906",
            "Avatar": "https://secure.gravatar.com/avatar/7c85cc31a189ddac41d391a7432d2985eda192186d1795e346ee26f1f487bf46?s=80&d=identicon",
            "Created": "0001-01-01T00:00:00Z",
            "Updated": "0001-01-01T00:00:00Z"
        },
        "Closed": false,
        "Labels": null,
        "Merged": false,
        "Number": 11,
        "Source": "playbooks-branch-alertmanager-kube-prometheus-stack-alertmanager-2",
        "Target": "main",
        "Created": "2024-03-12T10:57:47.519Z",
        "Updated": "2024-03-12T10:57:47.519Z",
        "DiffLink": "",
        "MergeSha": "",
        "Assignees": null,
        "Mergeable": false,
        "Milestone": {
            "ID": 0,
            "Link": "",
            "State": "",
            "Title": "",
            "Number": 0,
            "DueDate": null,
            "Description": ""
        },
        "Reviewers": null,
        "Rebaseable": false,
        "MergeableState": ""
    },
    "logs": "2024-03-12T16:42:43+05:45 Patching main.yaml\n2024-03-12T16:42:43+05:45 Creating file d9de8762-8a70-40b3-b15b-e10ec556b184\n2024-03-12T16:42:46+05:45 committed(b7afbf2ec7b07043760c1dc06eeb14d412dc72cc) and pushed changes\n2024-03-12T16:42:46+05:45 creating pull request: https://gitlab.com/flanksource-aditya/test\nbranch: playbooks-branch-alertmanager-kube-prometheus-stack-alertmanager-2\nauthor: John Doe\n2024-03-12T16:42:49+05:45 created pull request: https://gitlab.com/flanksource-aditya/test/-/merge_requests/11",
    "links": [
        {
            "id": 11,
            "url": "https://gitlab.com/flanksource-aditya/test/-/merge_requests/11",
            "icon": "pr",
            "name": "chore: Adding pod alertmanager-kube-prometheus-stack-alertmanager-2"
        }
    ]
}

```